extensions/amp-video/amp-video.md

```diff
@@ -112,6 +112,10 @@ Determines whether the media element is allowed to have a remote playback UI suc
 
 The `muted` attribute is deprecated and no longer has any effect. The `autoplay` attribute automatically controls the mute behavior.
 
+##### rotate-to-fullscreen
+
+Automatically shows the video on fullscreen after the user rotates their device into landscape mode, if the video is visible. See [more details about this feature.](https://github.com/ampproject/amphtml/blob/master/spec/amp-video-interface.md#rotate-to-fullscreen)
```

Comment:
Suggest having "If the video is visible" at the start of the sentence, like: 

`If the video is visible, the video displays fullscreen after the user rotates their device into landscape mode. For more details, see the [Video in AMP spec](https://github.com/ampproject/amphtml/blob/master/spec/amp-video-interface.md#rotate-to-fullscreen).`